src/Utils/GalleryUtils.ts

```diff
@@ -108,8 +118,17 @@ export function reportAbuse(
     undefined,
     "Report Abuse",
     async () => {
-      const clearSubmitReportNotification = NotificationConsoleUtils.logConsoleProgress(
-        `Submitting your report on ${data.name} violating code of conduct`
+      dialogHost.showOkCancelModalDialog(
+        "Report Abuse",
+        `Submitting your report on ${data.name} violating code of conduct`,
```

Comment:
>Submitting your report on ${data.name} violating code of conduc [](start = 9, length = 63)

Just say: Submitting your report? Because unless our AVERT team validates, we cannot call it as for violating code of conduct?

Reply:
I'll note it down and have a PR with all string updates later on. Trying to check this in asap to get feedback.